packages/@cdktf/hcl2cdk/lib/coerceType.ts

```diff
@@ -6,6 +6,9 @@ import { logger } from "@cdktf/commons";
 import { AttributeType } from "@cdktf/provider-generator";
```

Comment:
Trying to consolidate all type related functions to this file rather than them living in expression.ts